.github/workflows/test.yml

```diff
@@ -40,16 +40,16 @@ jobs:
       run: |
         go build -v .
 
-
-# run acceptance tests in a matrix with Terraform core versions
   test:
-    name: Matrix Test
+    name: 'Acc. Tests (OS: ${{ matrix.os }} / TF: ${{ matrix.terraform }})'
     needs: build
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
```

Comment:
Not that it needs to be changed back, but it wouldn't surprise me if this provider needs the longer run time. It's testing is not the most optimized.